modules/ROOT/pages/implementing-apikit-4-for-soap.adoc

```diff
@@ -0,0 +1,12 @@
+= Implementing a SOAP Interface
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Create an API by using Studio and APIkit for SOAP to consume a remote WSDL API. Download this link:{attachmentsdir}/tshirt2.wsdl[WSDL tshirt2.wsdl] example to create a SOAP interface that accesses data resources and integrates the interface with backend processes that function as an API.
```

Comment:
```suggestion
Create an API by using Anypoint Studio and APIkit for SOAP to consume a remote WSDL API. Download this {attachmentsdir}/tshirt2.wsdl[WSDL tshirt2.wsdl example] to create a SOAP interface that accesses data resources and integrates the interface with backend processes that function as an API.
```

Reply:
I'm confused by the first sentence. Are you using Studio and APIkit to create an API that consumes a remote WSDL API?

Reply:
Yes! You're right!
Please, let me know if you need more info about this topic.
Thanks!